lib/ansible/modules/cloud/openstack/os_tempest_run.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/python
+
+# (c) 2017, Tal Shafir
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: os_tempest_run
+short_description: Runs Tempest
+description:
+    - Runs Tempest according to the configuration file in the given workspace
+
+version_added: "2.4"
+
+author: "Tal Shafir , @TalShafir"
+requirements: ["Tempest"]
+options:
+    workspace:
+        description:
+            The workspace as was configured in 'Tempest init <I(workspace)>'
```

Comment:
The `I(workspace)` will turn into italics, which I'm not sure if that's what you want in this case

https://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#formatting-options